Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1920,13 +1920,13 @@ WebsiteDataStoreParameters WebsiteDataStore::parameters()
 #endif
 
 #if PLATFORM(IOS_FAMILY)
+    if (auto directory = cookieStorageDirectory(); !directory.isEmpty())
```

Comment:
I don't understand this change, Those will always be empty now and be no-ops if !isPersistent(), so why move them out of the `if (isPersistent())` scope and then 1 check for 3 checks?

Reply:
Okay, will revert this part.